src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -49,21 +49,23 @@ using SchedulingKey = std::tuple<SchedulingClass, std::vector<ObjectID>, ActorID
 // This class is thread-safe.
 class CoreWorkerDirectTaskSubmitter {
  public:
-  CoreWorkerDirectTaskSubmitter(rpc::Address rpc_address,
-                                std::shared_ptr<WorkerLeaseInterface> lease_client,
-                                rpc::ClientFactoryFn client_factory,
-                                LeaseClientFactoryFn lease_client_factory,
-                                std::shared_ptr<CoreWorkerMemoryStore> store,
-                                std::shared_ptr<TaskFinisherInterface> task_finisher,
-                                ClientID local_raylet_id, int64_t lease_timeout_ms)
+  explicit CoreWorkerDirectTaskSubmitter(
+      rpc::Address rpc_address, std::shared_ptr<WorkerLeaseInterface> lease_client,
+      rpc::ClientFactoryFn client_factory, LeaseClientFactoryFn lease_client_factory,
+      std::shared_ptr<CoreWorkerMemoryStore> store,
+      std::shared_ptr<TaskFinisherInterface> task_finisher, ClientID local_raylet_id,
+      int64_t lease_timeout_ms,
+      std::function<Status(const TaskSpecification &, const gcs::StatusCallback &)>
+          &&actor_create_helper = nullptr)
```

Comment:
Seems a bit much to use an r-value reference here, we can just let the function be copied since it's not performance critical.